.github/workflows/push.yml

```diff
@@ -158,7 +158,8 @@ jobs:
       - name: Run code linters
         run: |
           make mod
-          make lint
+# TODO temporarily disabled due to https://keboola.atlassian.net/browse/KAC-167
+#          make lint
```

Comment:
Dneska začaly failovat lint akce v Githubu, lokálně mi to jede (viz. popis https://keboola.atlassian.net/browse/KAC-167). Zatím moc netuším co se děje, nic moc se mi vygooglit nepodařilo, tak se uchyluju k poslední možnosti co mi zbývá, dočasně to vypnout. 😕 Budu se to snažit zkoumat dál, ale potřebuju taky pokračovat v práci. Kdybys náhodou věděl co s tím, tak budu šťastnej za radu. 🙂 Já ani moc nevím co ten `metalinter` konkrétně dělá. Myslel jsem si že ho jen vypnu tady někde: https://github.com/keboola/keboola-as-code/blob/main/build/ci/golangci.yml ale zřejmě je to nějaký builtin nebo co.

Reply:
Pokud na tom ale nechceš trávit čas tak je to asi jedno, stačí když to budeš zatím pouštět lokálně.

Reply:
Cache mě napadla, zkoušel jsem vypnout tu `load-cache` akci (https://github.com/keboola/keboola-as-code/blob/8777cc0349752bd48da292d943349b393f14d785/.github/workflows/push.yml#L151-L154) ale to nepomohlo. Vidíš, ale to smazání přes API bych ještě zkusit mohl. 👍

Reply:
Nepomohlo, škoda. 😕 Ale díky, stálo to za pokus. 👍
![CleanShot 2022-07-12 at 11 47 56](https://user-images.githubusercontent.com/215660/178462478-522cfd29-cf3e-4339-b598-62717745cb9d.jpg)